javascript/javascript1/week1/housey-pricey/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+
+
+name = "Peter";
```

Comment:
Would be nice to define the variables first with `let`. For example:
```
let personName;
```

It's preferred to avoid generic variable names as `name`, `length`, `math`, `function` and others. Many of those and more are used in JavaScript as methods and objects operators. You can see a detailed list of variable names to avoid here - https://www.w3schools.com/js/js_reserved.asp